runtime/bootstrap/index.ts

```diff
@@ -151,7 +153,11 @@ export async function executeThunk<T extends FormulaSpecification>(
   },
   packBundlePath: string,
   packBundleSourceMapPath: string,
-): Promise<T extends SyncFormulaSpecification ? GenericSyncFormulaResult : PackFormulaResult> {
+): Promise<
+  T extends SyncUpdateFormulaSpecification ? GenericSyncUpdateResult
```

Comment:
Optional, it seems like this return type is actually the same as `PackFunctionResponse<T>` so could reuse your new named type if you want.

Reply:
I'll have to add some type imports in a shared location that doesn't yet contain any, but seems doable.  Done.